packages/components/src/Popover/Popover.tsx

```diff
@@ -50,26 +50,26 @@ export function Popover({
     },
   );
 
-  return (
-    open && (
-      <div
-        role="dialog"
-        ref={setPopperElement}
-        style={popperStyles.popper}
-        className={classes.popover}
-        {...attributes.popper}
-      >
-        <div className={classes.dismissButton}>
-          <ButtonDismiss onClick={onRequestClose} ariaLabel="Close dialog" />
-        </div>
-        {children}
-        <div
-          ref={setArrowElement}
-          className={classes.arrow}
-          style={popperStyles.arrow}
-        />
+  return open ? (
+    <div
+      role="dialog"
+      ref={setPopperElement}
+      style={popperStyles.popper}
+      className={classes.popover}
+      {...attributes.popper}
+    >
+      <div className={classes.dismissButton}>
+        <ButtonDismiss onClick={onRequestClose} ariaLabel="Close dialog" />
       </div>
-    )
+      {children}
+      <div
+        ref={setArrowElement}
+        className={classes.arrow}
+        style={popperStyles.arrow}
+      />
+    </div>
+  ) : (
+    <></>
```

Comment:
@robbyjobber I believe what we would normally do here is something like this.

```tsx
return (
  <>
    {open && (
      ...
    )}
  </>
)
```

This way, we are always returning _at least_ a fragment, and we don't end up with a weird ternary in our component.

Reply:
oh makes sense!